analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -53,6 +55,9 @@
     <Content Include="TestResources\**\*">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="TestFramework\Razor\EmptyProject\**\*">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
I would go with Always and then we can never have a problem with this. WDYT?

Reply:
I agree with you. In the past, I have always been using `Always`. As you say, you avoid issues related to wrongly cached files.
I guess  `TestCases\**\*"` was set to `PreserveNewest` since that folder contains a lot of files, and only copying when needed yields tangible performance benefits when building.
However, `TestResources` is not as big, and yet, it is set to `PreserveNewest`.
In any case, `TestFramework\Razor\EmptyProject` is even smaller, so `Always` seems really OK.
If you confirm, I make the change.